code/game/objects/items/tools/screwdriver.dm

```diff
@@ -25,54 +25,42 @@
 	drop_sound = 'sound/items/handling/screwdriver_drop.ogg'
 	pickup_sound =  'sound/items/handling/screwdriver_pickup.ogg'
 	sharpness = SHARP_POINTY
-	var/random_color = TRUE //if the screwdriver uses random coloring
+	/// If the item should be assigned a random color
+	var/random_color = TRUE
+	/// List of possible random colors
 	var/static/list/screwdriver_colors = list(
-		"blue" = rgb(24, 97, 213),
```

Comment:
Is there some reason you changed from the rgb() format? These just output a color hex like you've manually done here.

Reply:
I just wanted to be consistent with other works and I also think RGB did not show color icon preview in VSC for me.

Reply:
ah fair enough